src/Config.php

```diff
@@ -16,7 +16,7 @@ class Config
      */
     public function __construct(array $items = null, string $base_config_path = null)
     {
-        $this->items = require($base_config_path ?: __DIR__ . "./../config/msg91.php");
+        $this->items = require($base_config_path ?: __DIR__ . "/../config/msg91.php");
```

Comment:
There is one more change we can put here: `DIRECTORY_SEPARATOR`

Instead of slash, lets use this global value to support other OS. 

```diff
- $this->items = require($base_config_path ?: __DIR__ . "/../config/msg91.php");
+ $this->items = require($base_config_path ?: __DIR__ . DIRECTORY_SEPARATOR .  
+   join(DIRECTORY_SEPARATOR, array('..', 'config', 'msg91.php')
+ ));
```

Feel free to improve the formatting if required. You might want to test the change in your existing project as well.

Reply:
When using this in my existing project, i receive this error: 

![image](https://github.com/user-attachments/assets/b6234c8f-898a-4ada-9d09-3cfd25d0b50e)


I used this code: 

```php
        $this->items = require($base_config_path ?: __DIR__ . DIRECTORY_SEPARATOR . + join(DIRECTORY_SEPARATOR, array('..', 'config', 'msg91.php')));
```

Reply:
You have added a plus before `join`.  Please remove it.

Reply:
Sorry it was my mistake. Now it is working fine!

removing the `+`, do works!